lib/flyover_comments/authorization.rb

```diff
@@ -11,6 +11,16 @@ def can_delete_flyover_comment?(comment, user)
       end
     end
 
+    def can_view_flyover_comment?(comment, user)
```

Comment:
Check is this correct? Same permissions as create, though that might not be the case.